app/models/case_log.rb

```diff
@@ -35,7 +35,8 @@ class CaseLog < ApplicationRecord
   belongs_to :owning_organisation, class_name: "Organisation", optional: true
   belongs_to :managing_organisation, class_name: "Organisation", optional: true
   belongs_to :created_by, class_name: "User", optional: true
-
+  belongs_to :scheme, required: false
```

Comment:
`required` option is deprecated. `optional: true` as above is recommended now I think (https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/belongs_to)